dhcpv6/client6/iputils.go

```diff
@@ -1,8 +1,10 @@
-package dhcpv6
+package client6
```

Comment:
I would leave this in `dhcpv6` since it may also be used by the `server` code.

Reply:
good point - there are several things that may go to a separate common package, but I guess `dhcpv6` is still a better option than adding another one